docs/0.7/references/api/openapi.yml

```diff
@@ -346,27 +346,212 @@ paths:
           schema:
             type: string
       responses:
-        200:
+        '200':
           description: Successfully retrieved the requested circuit
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/Circuit"
-        401:
+        '401':
           description: The client is unauthorized
-        404:
+        '404':
           description: The requested circuit was not found
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-        500:
+        '500':
           description: An internal server error occurred
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
 
+  /authorization/assignments:
+    parameters:
+      - $ref: "#/components/parameters/auth"
+      - $ref: "#/components/parameters/protocol_version"
+    get:
+      tags:
+        - Authorization
+      summary: Get all assignments
+      description: |
+        Gets all assignments.
+
+        This endpoint requires the permission "authorization.rbac.read".
+
+      parameters:
+        - $ref: "#/components/parameters/paging"
+      responses:
+        200:
```

Comment:
should these response codes also be in single quotes like the ones above?

Reply:
This is directly copied from splinter using `just copy-docs`. If their are issues they should be fixed their and then copied over again. Same for the below item. The reason I fixed the line wrap in the playlist man page is because that would have failed lint (there is a PR up to fix it in splinter as well).

Reply:
got it, sounds good